src/config.rs

```diff
@@ -829,6 +896,7 @@ impl Default for DbConfig {
             writecf: WriteCfConfig::default(),
             lockcf: LockCfConfig::default(),
             raftcf: RaftCfConfig::default(),
+            versioncf: VersionCfConfig::default(),
```

Comment:
You need to consider the existing cluster, there are only 4 cfs, you should create it before open it. Please ref to https://github.com/tikv/tikv/blob/master/components/engine/src/rocks/util/mod.rs#L125

Reply:
it would create the need diff cfs,  see https://github.com/tikv/tikv/blob/master/components/engine/src/rocks/util/mod.rs#L215